### DISCUSSION TITLE
青衣池吃满保底歪了啊啊啊啊！

### DISCUSSION THREAD

**Discussion by JobNumber-9527** [share121/inter-knot#2576] [其他]
被忽悠抽1命青衣，吃满保底歪了个狼哥啊啊！大保底快满了也没出！我的菲林啊！都非走了！<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >
![IMG_3663](https://github.com/user-attachments/assets/5814d7c0-ff86-41ae-b24e-378853d11fff)
![IMG_3664](https://github.com/user-attachments/assets/63e6f651-5a64-4028-826b-46d506e43055)

**Comment by UEG-LmR** [comment]
没事我出青衣了，电属性拿枪的。

**Reply by xiaoxi4233** [reply]
我们俩的青衣一样
